testing/citest/tests/appengine_gcs_pubsub_test.py

```diff
@@ -322,15 +322,15 @@ def trigger_deploy_pipeline(self):
          [jp.DICT_MATCHES({'name': jp.STR_EQ(group_name)})]
        )))
 
-    executions_path = 'executions?pipelineConfigIds={}&limit=1&statuses=SUCCEEDED'.format(self.__pipeline_id)
+    executions_path = 'executions?pipelineConfigIds={}'.format(self.__pipeline_id)
     return st.OperationContract(
```

Comment:
ok, but if you fail and retry, would there still be something there and would that cause the pipeline status to think it finished?